cloud-movies-be/lib/api-gw/api-gw-stack.ts

```diff
@@ -135,5 +159,13 @@ export class ApiGwStack extends cdk.Stack {
         validateRequestParameters: true,
       },
     });
+
+    const searchMoviesLambdaIntegration = new apigateway.LambdaIntegration(searchMoviesFn);
+    const searchMoviesResource = api.root.addResource("search-movies");
+    searchMoviesResource.addMethod("POST", searchMoviesLambdaIntegration, {
```

Comment:
This can also be a GET request, and we can provide query parameters to search the movies. It will get along better with REST principles.